rocketchat/templates/podmonitor.yaml

```diff
@@ -21,7 +21,7 @@
 # Note: This approach trades off the automatic filtering provided by ServiceMonitors in
 # favor of comprehensive monitoring coverage across all pod states.
 */}}
-{{- if and .Values.podMonitor.enabled .Values.prometheusScraping.enabled}}
+{{- if and .Values.podMonitor.enabled .Values.prometheusScraping.enabled (.Capabilities.APIVersions.Has "monitoring.coreos.com/v1") }}
```

Comment:
Same comment as before. Double checking as I don't quite remember. Generally it's two operands not 2+.